source/fundamentals/databases-collections.txt

```diff
@@ -0,0 +1,185 @@
+=========================
+Databases and Collections
+=========================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+MongoDB organizes data into a hierachy of the following levels:
+
+- databases
+
+- collections
+
+- documents
+
+Databases are the top level of data organization in a MongoDB instance.
+Databases are organized into collections, which contain documents.
+Documents contain literal data like strings, numbers, and dates as well
+as other (embedded) documents.
+
+Access a Database
+-----------------
+
+Use the :java-sync-api:`getDatabase()
+<com/mongodb/client/MongoClient.html#getDatabase(java.lang.String)>` method of
+a ``MongoClient`` instance to access a ``MongoDatabase`` in a MongoDB
+instance.
+
+The following example accesses a database named "testDatabase":
+
+.. code-block:: java
+
+   MongoDatabase database = mongoClient.getDatabase("testDatabase");
+
+Access a Collection
+-------------------
+
+Use the :java-sync-api:`getCollection()
+<com/mongodb/client/MongoDatabase.html#getCollection(java.lang.String)>`
+method of a ``MongoDatabase`` instance to access a
+``MongoCollection`` in a database of your connected MongoDB instance.
+
+The following example accesses a collection named "testCollection" from
+a ``MongoDatabase``:
+
+.. code-block:: java
+
+   MongoDatabase database = mongoClient.getDatabase("testDatabase");
+   MongoCollection<Document> collection = database.getCollection("testCollection");
+
+.. note::
+
+   If the provided collection name does not already exist in the database,
+   MongoDB implicitly creates the collection when you first insert data
+   into that collection.
+
+Create a Collection
+-------------------
+
+Use the :java-sync-api:`createCollection()
+<com/mongodb/client/MongoDatabase.html#createCollection(java.lang.String,com.mongodb.client.model.CreateCollectionOptions)>`
+method of a ``MongoDatabase`` instance to create a collection called
+"exampleCollection" in a database of your connected MongoDB instance:
+
+.. code-block:: java
+
+   database.createCollection("exampleCollection");
+
+You can specify collection options like maximum size and document
```

Comment:
Not sure if we want to mention maximum size anymore.  Aren't capped collections out of favor, and TTL indexes in favor?

Reply:
Unfortunately I'm not sure what's in vogue right now, but as they haven't been deprecated I think they still serve as a reasonable example of ``CreateCollectionOptions``.

That being said, if TTL indexes are an important concept, I'd be happy to add them to the indexes page (and presumably add an admonition to the capped indexes section pointing to TTL indexes) once we create that page!

Reply:
I think capped collections are not in vogue (they don't work sharded, for instance).  

Seems like document validation is enough of an example for create collection options

Reply:
Removed the capped collections example.